src/Field.php

```diff
@@ -27,6 +27,11 @@ class Field implements IteratorAggregate
     protected $required = false;
 
     /**
+     * @var string
+     */
+    protected $type = 'string';
```

Comment:
This should likely be `null` as the default, considering we don't have a way to specify the field type in the UI at this time.